.github/workflows/js-and-android-test.yml

```diff
@@ -40,9 +40,15 @@ jobs:
                       - 'feature-utils/**'
                       - 'platform/feature-api/**'
                       - 'platform/podjs/**'
+                      swift:
+                      - 'platform/ios/**'
             - name: Lint core and features
               if: steps.changes.outputs.js == 'true'
               run: ./build.js lint
+            - name: Lint Swift
```

Comment:
Not sure if we want to do it in this PR, but also the name of the workflow file needs to be updated to include also iOS.

Reply:
Not really doing any iOS stuff, other than linting Swift... But we might want to add "linting" to the name... somewhere (kotlint is done elsewhere, for no reason)

Reply:
No strong preference. I was mostly thinking of bundling related jobs in the same workflow. But you know more about the whole CI setup, so if it is better to have it running on Ubuntu let's keep it as is.